### PR TITLE
prepared a more meaningful example of tvtk usage in Mayavi

### PR DESCRIPTION
**Pull request by marekyggdrasil**
Hi all!

I have been struggling with `tvtk` usage in Mayavi, detail on why are provided in [this question](https://stackoverflow.com/questions/64639371/how-to-rotate-actor-on-tvtk-rendered-scene-with-mayavi-in-python/64656357#64656357) (which is answered now).

I figured a more meaningful [example](https://docs.enthought.com/mayavi/mayavi/auto/example_tvtk_in_mayavi.html#example-tvtk-in-mayavi) could be useful. The previous scene was:

snapshot0

And now it is:

snapshot1

I believe it is better because:

1. This example teaches how to place glyphs in different orientation.
2. Teaches how to render a cylinder and adjust its resolution.
3. Cylinder is more visually meaningful than a sphere because of lack of spherical symmetry - orientation matters.
4. It is more visually meaningful if glyphs have different colours, it makes it provides a one more criterion helping to associate them with objects in the code.

If you need some more adjustments let me know and if you are not interested in it just close the PR. Thanks!

### CODE REVIEW

**Comment by marekyggdrasil** [comment]
I realised I was specifying the glyph's position by shifting its center... It is fixed now.

snapshot

File: examples/mayavi/mlab/tvtk_in_mayavi.py [hunk 1]
```diff
@@ -42,17 +42,17 @@
...
-p = tvtk.Property(opacity=0.3, color=(1, 0, 0))
-sphere_actor2 = tvtk.Actor(mapper=sphere_mapper2, property=p)
-v.scene.add_actor(sphere_actor2)
+# Create a cylinder
+cylinder = tvtk.CylinderSource(center=(0, 0, 0), radius=0.2, resolution=16)
+cylinder_mapper = tvtk.PolyDataMapper()
+configure_input_data(cylinder_mapper, cylinder.output)
+cylinder.update()
+p = tvtk.Property(opacity=0.3, color=(0, 0, 1))
+cylinder_actor = tvtk.Actor(position=(7, 0, 1), mapper=cylinder_mapper, property=p, orientation=(90, 0, 90))
```
Comment (prabhuramachandran) [inline examples/mayavi/mlab/tvtk_in_mayavi.py:54]: Could you please wrap this to 80 cols?

**Review by prabhuramachandran — CHANGES_REQUESTED** [review]
Thank you, looks good, please make the minor change I suggest and I will merge this.

**Comment by prabhuramachandran** [comment]
@marekyggdrasil -- please also rebase to current master.  Thank you.

File: examples/mayavi/mlab/tvtk_in_mayavi.py (hunk 1 above)
Comment (marekyggdrasil) [reply examples/mayavi/mlab/tvtk_in_mayavi.py]: Sure thing!

Before the change:
```
$ pylint --max-line-length=80 tvtk_in_mayavi.py 
************* Module tvtk_in_mayavi
tvtk_in_mayavi.py:54:0: C0301: Line too long (108/80) (line-too-long)

------------------------------------------------------------------
Your code has been rated at 9.71/10 (previous run: 9.71/10, +0.00)
```
After the change:
```
$ pylint --max-line-length=80 tvtk_in_mayavi.py 

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 9.71/10, +0.29)
```

**Comment by marekyggdrasil** [comment]
@prabhuramachandran thanks for reviewing! I wrapped the lines to match the requested linting style and also merged the most recent master. No conflicts!

**Review by prabhuramachandran — APPROVED** [review]
LGTM, thanks.

**Pull request merged**